### PR TITLE
Merge -multicert/-cert and -multikey/-key into new flags -certs and -keys

### PR DESCRIPTION
**Pull request by jkroll-cf**
This makes configuration easier, as the same flag is used regardless of how many keys/certs are being provided. Also, it fixes a bug where -cert and -key would not have worked.

### CODE REVIEW

File: redoctober.go [hunk 1]
```diff
@@ -204,7 +204,7 @@ const usage = `Usage:
 example:
 redoctober -vaultpath diskrecord.json -addr localhost:8080 -cert cert.pem -key cert.key
```
Comment (grittygrease) [inline redoctober.go]: Is this still valid?

Comment (jkroll-cf) [inline redoctober.go]: Nope. Fixed.

**Comment by grittygrease** [comment]
Note that this is a breaking change for current implementations.

**Comment by grittygrease** [comment]
Please update README.md and redoctober.service.

**Comment by jkroll-cf** [comment]
Have updated the README and the example .service file and confirmed with grep that these are the only remaining places with the old flags.

**Comment by lziest** [comment]
LGTM

**Pull request merged**